ztp/gitops-subscriptions/argocd/example/policygentemplates/common-ranGen.yaml

```diff
@@ -3,11 +3,12 @@ apiVersion: ran.openshift.io/v1
 kind: PolicyGenTemplate
 metadata:
   name: "common"
-  namespace: "ztp-common"
+  namespace: "ztp-common-latest"
```

Comment:
I think we want to change the name rather than the namespace. This has a couple of impacts to the user:
1. If you change the namespace you will need to copy any hub side templating resources (eg configmap) to the new namespace. This results in duplication or work by the user to remove cluster-abc from one configmap and into another when they change the binding version.
2. TALM and ACM both have restrictions on two policies with the same name but different namespaces if they get bound to the same cluster. With versioning like this it should not happen but I think we should use unique names to stay far away from this limititation. TALM's limitation is in the policy name, ACM's is in the policy-template name.

Reply:
/test e2e-gcp-ovn

Reply:
okay, done